src/common.js

```diff
@@ -1,3 +1,7 @@
+/* global Nimiq */
+/* global KeyStore */
+/* global RpcServer */
+
```

Comment:
I would prefer a one-liner (global Nimiq, Keystore...)

Reply:
This could become a very long line, e.g. here: https://github.com/nimiq/keyguard-next/pull/25/files#diff-941ff894131b0eb2fea1412f106d8d2eR1

This would break with the 120 character limit.

I like them better under each other, just like classic import statements.

Reply:
Yes, multiple lines is better